pkgs/development/python-modules/tensorflow/bin.nix

```diff
@@ -102,10 +94,10 @@ in buildPythonPackage {
     (
       cd unpacked/tensorflow*
       # relax too strict versions in setup.py
-      patch -p 1 < ${patch}
+      patch -p 1 < ${./relax-dependencies.patch}
```

Comment:
To make things more overridable one can do
```
foopatch = ./foo.patch;
preConfigure = ''
  patch -p 1 < "$foopatch"
'';
```

Reply:
Okay, I revert that. Thanks!